libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/PluggableColumnBarCharts.tsx

```diff
@@ -245,8 +244,12 @@ export class PluggableColumnBarCharts extends PluggableBaseChart {
             }
         }
 
-        const finalStacks = [...stacks, ...possibleStacks].slice(0, stackByMaxItemCount);
-        return [views, finalStacks];
+        if (!stacks.length && measures.length <= 1) {
+            const finalStacks = [...stacks, ...possibleStacks].slice(0, stackByMaxItemCount);
+            return [views, finalStacks];
```

Comment:
This is weird...you have a condition that checks if _at least_ one measure is in the buckets but then you return just views and stacks. Cannot it be a problem? You are taking also measures now from the getBucketItems function.

Reply:
Yes it is mistake now fixed